src/show/components/Navbar/index.js

```diff
@@ -1,33 +1,69 @@
-import React from "react";
+import { React, useState } from "react";
 import { Link, useNavigate } from "react-router-dom";
-
+import { XMarkIcon } from '@heroicons/react/24/solid';
 import { Cart } from "../../assets/svg";
+import MenImage from "../../assets/img/men.png";
+import WomenImage from "../../assets/img/women.jpg";
+import KidsImage from "../../assets/img/kid.png";
```

Comment:
i told you something about imports. please follow only that. I won't tell you again